vividus-engine/src/test/java/org/vividus/expression/NormalizingArgumentsExpressionTests.java

```diff
@@ -42,7 +42,6 @@ void shouldEvaluateExpression(String expression, String expectedResult)
     }
 
     private static class TestExpressionProcessor extends AbstractExpressionProcessor<String>
-            implements NormalizingArguments
```

Comment:
since no such interface is available anymore, the test class should be renamed to test utility class (or completely removed)